src/iomusxml.cpp

```diff
@@ -2708,9 +2708,15 @@ void MusicXmlInput::ReadMusicXmlNote(
         pugi::xpath_node startTie = notations.node().select_node("tied[@type='start']");
         pugi::xpath_node endTie = notations.node().select_node("tied[@type='stop']");
         if (endTie) { // add to stack if (endTie) or if pitch/oct match to open tie on m_tieStack
-            m_tieStopStack.push_back(note);
+            if (note->GetPname() == m_tieStack.back().second->GetPname() && note->GetOct() == m_tieStack.back().second->GetOct()) {
```

Comment:
Just to make sure: don't you need to check that m_tieStack is not empty before accessing `.back()`?

Reply:
On Friday I don't need that, but on Monday I don't feel so sure about that anymore … ;-)